kwave/executor.py

```diff
@@ -85,7 +85,7 @@ def parse_executable_output(output_filename: str) -> dotdict:
         with h5py.File(output_filename, "r") as output_file:
             sensor_data = {}
             for key in output_file.keys():
-                sensor_data[key] = output_file[f"/{key}"][0].squeeze()
+                sensor_data[key] = output_file[f"/{key}"][:].squeeze()
```

Comment:
Why use this indexing since `x[:]` is equivalent to `x`, afik.

If I remember correctly the `x[0]` was to remove a 1 dimension in the zeroeth index position. What issue does this change address?

Reply:
Yes, the the 2d values are of shape (1, x, y), but 3d values (e.g. if you record p_final) are of shape (x, y, z), so their first axis was removed.

Reply:
ok great.